### PR TITLE
Update Elsys name

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
Change ElektronikSystem to Elsys.

#### Summary

...

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/index.yaml [hunk 1]
```diff
@@ -1691,7 +1691,7 @@ vendors:
     website: https://www.innotas-elektronik.de/en-gb
 
   - id: elsys
-    name: ElektronikSystem i Ume
+    name: Elsys
```
Comment (benolayinka) [inline vendor/index.yaml:1694]: Why are they in here twice @Jaime-Trinidad @johanstokking ? Can't we delete the second?

Comment (johanstokking) [reply vendor/index.yaml:1694]: Yes that is wrong. This one can and should be removed

**Comment by Jaime-Trinidad** [comment]
@benolayinka the second Elsys name now is deleted and profile files now contain elsys vendor Id.

File: vendor/elsys/elsys-as923-profile.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Vendor profile ID, can be freely issued by the vendor
 # This vendor profile ID is also used on the QR code for LoRaWAN devices, see
 # https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
-# vendorProfileID:
+vendorProfileID: 132
```
Comment (johanstokking) [inline vendor/elsys/elsys-as923-profile.yaml:4]: Profiles should have a unique number within the vendor space.

File: vendor/index.yaml [hunk 3]
```diff
@@ -1690,9 +1690,9 @@ vendors:
     name: Innotas Elektronik
     website: https://www.innotas-elektronik.de/en-gb
 
-  - id: elsys
-    name: ElektronikSystem i Ume
-    website: https://www.elsys.se/en/
+  #  - id: elsys
+  #    name: Elsys
+  #    website: https://www.elsys.se/en/
```
Comment (johanstokking) [inline vendor/index.yaml:1695]: Just delete it

**Review by johanstokking — APPROVED** [review]

**Pull request merged**